.pre-commit-config.yaml

```diff
@@ -2,15 +2,15 @@
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
   - repo: https://github.com/PyCQA/isort
-    rev: 5.12.0
+    rev: 5.13.2
```

Comment:
In my opinion this should be updated in different PR because is adding too much changes that are not related with the focus of the PR.

Reply:
Sorry, I was having an error with the current version

Reply:
Pre-commit was already updated but I didn't rebased